### PR TITLE
Mo/3.1.0 support

### PR DESCRIPTION
**Pull request by MarcoOl94**
Added new serialization/deserialization for the withdrawalEpochLength that includes also the sidechain version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitcore-lib-zen",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (otoumas) [inline package.json:3]: Maybe we should increase the MINOR version? 
According to [SemVer2](https://semver.org/) the PATCH version should be bumped if it is a bug. So the questions we need to ask ourselves is: 
1. Is this a bug?
2. Is this backwards compatible
Response to this questions:
1: yes --> PATCH
1:No, 2:yes --> MINOR
1:No, 2:NO --> MAYOR

**Review by otoumas — APPROVED** [review]
LGTM See comments

File: package.json (hunk 1 above)
Comment (MarcoOl94) [reply package.json:3]: This was changed accordingly to what @cronicc suggested.

Comment (otoumas) [reply package.json:3]: Ok so we consider it as a Bug then 😄 !
I'm going to start a campaign for everyone to fully understand SemVer versioning (starting by myself 😉 ). 
We have so many applications going on that we need to fully understand how to increase versions.

**Review by cronicc — APPROVED** [review]

**Pull request merged**